storagenode/satellites/satellites.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellites
+
+import (
+	"context"
+	"time"
+
+	"storj.io/storj/pkg/storj"
+)
+
+//Status refers to the state of the relationship with a satellites
+type Status = int
+
+const (
+	//Unexpected status should not be used for sanity checking
+	Unexpected Status = iota
+	//Normal status reflects a lack of graceful exit
+	Normal
+	//Exiting reflects an active graceful exit
+	Exiting
+	//ExitSucceeded reflects a graceful exit that succeeded
+	ExitSucceeded
+	//ExitFailed reflects a graceful exit that failed
+	ExitFailed
+)
+
+//ExitProcess contains the status of a graceful exit
+type ExitProcess struct {
+	SatelliteID       storj.NodeID
+	InitiatedAt       *time.Time
+	FinishedAt        *time.Time
```

Comment:
nit - I don't think these need to be pointers.

Reply:
They do because they're nullable in the DB... the SQL libs do all the work if they're nullable.